ruff.toml

```diff
@@ -0,0 +1,35 @@
+target-version = "py313"
+
+[lint]
+extend-select = [
+    "B",    # flake8-bugbear
+    "BLE",  # flake8-blind-except
+    "C4",   # flake8-comprehensions
+    "DJ",   # flake8-django
+    "E",    # pycodestyle errors
+    "F",    # pyflakes
+    "I",    # isort
+    "INT",  # flake8-gettext
+    "PIE",  # flake8-pie
+    "PGH",  # pygrep-hooks
+    "S",    # flake8-bandit
+    "SIM",  # flake8-simplify
```

Comment:
Most of these I had never heard of but solve things that semi-regularly come up in code review I do.

I certainly will be adopting this ruff config ❤️